scripts/apply-wmchat-consent-live.Tests.ps1

```
@@ -0,0 +1,339 @@
+        $env:WMCHAT_LIVE_DATABASE_URL = $DatabaseUrl
+        $env:WMCHAT_LIVE_PGSSLROOTCERT = $Fixture.Ca
+        $env:PATH = "$(Split-Path -Parent $FakePsql);$($env:PATH)"
+        $env:WMCHAT_TEST_STATE_FILE = $Fixture.State
+        $env:WMCHAT_TEST_LOG = $Fixture.Log
+        $env:WMCHAT_TEST_MUTATION_FAIL = $(if ($MutationFail) { "1" } else { "0" })
+        $env:WMCHAT_TEST_DRIFT = $(if ($RemoteDrift) { "1" } else { "0" })
+        $env:WMCHAT_TEST_POST_READ_FAIL = $(if ($PostReadFail) { "1" } else { "0" })
+        $env:WMCHAT_TEST_BASE_MD5 = $BasePayloadMd5
+        $env:WMCHAT_TEST_GRANT_MD5 = $GrantPayloadMd5
+        if ($PoisonLibpq) { $env:PGHOSTADDR="203.0.113.5"; $env:PGSSLMODE="disable"; $env:PGPASSFILE="C:\attacker\.pgpass" }
+        $Args = @("-NoProfile","-ExecutionPolicy","Bypass","-File",$Fixture.Script,"-ReleaseWorktree",$Fixture.Repo,"-ReleaseCommit",$Fixture.Commit)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'Args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)